### PR TITLE
Add initial docs section for FileVault

### PR DESCRIPTION
**Pull request by headmin**

### CODE REVIEW

File: docs/apps/mdm.md [hunk 1]
```diff
@@ -211,6 +211,45 @@ These configurations are typically applied by assigning a dedicated blueprint du
...
+Zentral manages FileVault settings for full disk encryption on macOS devices via MDM. Using a dedicated configuration, it allows the creation and assignment of an individual FileVault configuration to one or more MDM Blueprints. This approach provides centralized control over FileVault application, user experience, and key management, ensuring compliance with the organization’s data encryption policies.
+
+Enforcing FileVault during the Setup Assistant is supported starting from macOS 14.4 and later.
+
+### FileVault Key Escrow
+
+FileVault key escrow ensures that Personal Recovery Keys (PRKs) are securely stored and accessible:
+
+- **Escrow**: Zentral automatically escrows the PRK when FileVault is enabled and enforced via MDM on a device. Zentral deploys the certificate required to asymmetrically encrypt the recovery key for escrow.
+- **Authorized Retrieval**: The PRK can be accessed through the Zentral interface for device recovery, with each retrieval action audited in the *FileVault PRK Viewed* log for the device.
```
Comment (np5) [inline docs/apps/mdm.md:225]: And the API

File: docs/apps/mdm.md [hunk 2]
```diff
@@ -211,6 +211,45 @@ These configurations are typically applied by assigning a dedicated blueprint du
...
+
+Zentral manages FileVault settings for full disk encryption on macOS devices via MDM. Using a dedicated configuration, it allows the creation and assignment of an individual FileVault configuration to one or more MDM Blueprints. This approach provides centralized control over FileVault application, user experience, and key management, ensuring compliance with the organization’s data encryption policies.
+
+Enforcing FileVault during the Setup Assistant is supported starting from macOS 14.4 and later.
+
+### FileVault Key Escrow
+
+FileVault key escrow ensures that Personal Recovery Keys (PRKs) are securely stored and accessible:
+
+- **Escrow**: Zentral automatically escrows the PRK when FileVault is enabled and enforced via MDM on a device. Zentral deploys the certificate required to asymmetrically encrypt the recovery key for escrow.
```
Comment (np5) [inline docs/apps/mdm.md:224]: The individual certificates. One per device.

**Review by np5 — CHANGES_REQUESTED** [review]
Only 2 little things.

File: docs/apps/mdm.md (hunk 2 above)
Comment (headmin) [reply docs/apps/mdm.md]: @np5 see latest commit `5921512`

**Review by np5 — APPROVED** [review]
LGTM

**Pull request merged**